helix-view/src/editor.rs

```diff
@@ -686,6 +686,14 @@ pub enum Action {
     VerticalSplit,
 }
 
+/// Error thrown on failed document closed
+pub enum CloseError {
+    /// Document doesn't exist
+    DoesntExist,
```

Comment:
Minor nit: could we name this `DoesNotExist` instead?

Reply:
Or just NotExist